storage/innobase/xtrabackup/src/backup_copy.cc

```diff
@@ -1584,7 +1584,7 @@ bool backup_start(Backup_context &context) {
   xb_mysql_query(mysql_connection, "FLUSH NO_WRITE_TO_BINLOG BINARY LOGS",
                  false);
 
-  auto log_status = log_status_get(mysql_connection);
+  context.log_status = log_status_get(mysql_connection);
```

Comment:
Do we need this static global variable at all? I think log_status_get() is disgrace. Returning a global variable :)
AFAIK, even after this change, there will be two log_status. backup_context.log_status and the global log_status.

Will there be two copies of files in these two variables?

Reply:
Yes, in the end we had two copies. Adjusted now to use log_status global variable.